web/src/pages/semantic.tsx

```diff
@@ -1,25 +1,27 @@
 import { type NextPage } from "next";
 import React, { useState } from "react";
-import { API_URL } from "../settings";
+import { API_URL, initialQuestions } from "../settings";
 import type { Followup } from "../types";
 import Page from "../components/page";
 import { SearchBox } from "../components/searchbox";
 
+const randomQuestion = () =>
+  initialQuestions[Math.floor(Math.random() * initialQuestions.length)] || "";
+
 const ignoreAbort = (error: Error) => {
   if (error.name !== "AbortError") {
     throw error;
   }
 };
 
 const Semantic: NextPage = () => {
+  const [query, setQuery] = useState(randomQuestion());
```

Comment:
you might want functional init instead of executing the an expression on every render and then ignore the value:

```suggestion
  const [query, setQuery] = useState(randomQuestion);
```
or 
```suggestion
  const [query, setQuery] = useState(() => randomQuestion());
```